docs/dev_guide/custom_field.rst

```diff
@@ -43,3 +43,4 @@ The rest of this section will walk you through these steps.
    custom_field/select_vocab_terms
    custom_field/manual_field_creation
    custom_field/tripal_field_generator
+   custom_field/custom_widget
```

Comment:
I would actually put this above field_generator, after field creation.

Reply:
Ok, I'm convinced. I hesitated myself on the order

Reply:
yeah i get that.  For me the idea is fields generator doesnt teach you anything, so you should only use the convenience once you've tried to learn the basics of fields.